docs/index.rst

```diff
@@ -110,7 +110,8 @@ https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
 
 The Python bindings also provide some additional configuration properties:
 
-* **DEPRECATED** ``default.topic.config``: value is a dict of client topic-level configuration
+* **DEPRECATED**: topic configurations should be specified in the global configuration **
```

Comment:
the trailing double splats look wrong?

Reply:
The line now reads as if "topic confiugration (not configruations!) should be specified in the global config .." is DEPRECATED.

I would use something like:
"default.topic.config: value is a dict of client topi-level configuration properties that are applied to all used topics for the instance. **DEPRECATED**: topic configuration should now be specified in the global top-level configuration

Reply:
makes sense